build.gradle

```diff
@@ -38,9 +38,15 @@ repositories {
 // PROJECT IDENTIFICATION
 // -----------------------------------------------------------------------------
 
+def version = 'git describe --dirty'.execute()
+version.waitFor()
+if (version.exitValue() !=0) {
+    throw new GradleException("Git describe failed:\n${cmd.err.text}")
+}
+
```

Comment:
We should be using our `gradle-baseline` plugin instead which standardises this. Will udpate.